stack/indexer/rpm/build_package.sh

```diff
@@ -8,7 +8,7 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-set -e
+set -ex
```

Comment:
Debug element?

Reply:
Additional information generator, I think it's required.